internal/services/recoveryservices/backup_protected_vm_resource_test.go

```diff
@@ -277,6 +317,29 @@ resource "azurerm_backup_protected_vm" "test" {
   recovery_vault_name = azurerm_recovery_services_vault.test.name
   source_vm_id        = azurerm_virtual_machine.test.id
   backup_policy_id    = azurerm_backup_policy_vm.test.id
+
+  disk_exclusion {
+    disk_lun_list     = [0]
+    is_inclusion_list = true
+  }
+}
+`, r.base(data))
+}
+
+func (r BackupProtectedVmResource) updateDiskExclusion(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_backup_protected_vm" "test" {
+  resource_group_name = azurerm_resource_group.test.name
+  recovery_vault_name = azurerm_recovery_services_vault.test.name
+  source_vm_id        = azurerm_virtual_machine.test.id
+  backup_policy_id    = azurerm_backup_policy_vm.test.id
+
+  disk_exclusion {
+    disk_lun_list     = [0, 1]
+    is_inclusion_list = false
+  }
```

Comment:
so looking at this, i think a better ux might be

```suggestion
  include_disk_luns  = [0, 1]
```
and have another property
```suggestion
  exclude_disk_luns  = [0, 1]
```
simpler and otherwise we would have excluded_disks.is_included which does not really make sense 🙃

Reply:
Thank you, fixed.